src/main/java/com/alipay/remoting/rpc/protocol/RpcHeartbeatTrigger.java

```diff
@@ -156,6 +157,33 @@ public void run(Timeout timeout) throws Exception {
                 }
             }, heartbeatTimeoutMillis, TimeUnit.MILLISECONDS);
         }
+    }
+
+    private void closeChannel(Connection connection) {
+        if (connection == null) {
+            return;
+        }
+
+        final Channel channel = connection.getChannel();
+        try {
+            if (channel != null) {
+                channel.close().addListener(new ChannelFutureListener() {
+
+                    @Override
+                    public void operationComplete(ChannelFuture future) throws Exception {
+                        if (logger.isInfoEnabled()) {
+                            logger.info(
+                                "Close the connection to remote address={}, result={}, cause={}",
+                                RemotingUtil.parseRemoteAddress(channel), future.isSuccess(),
+                                future.cause());
```

Comment:
add "cause by heartbeat loss" to message.

Reply:
done